source-code/ide-extension/src/main.ts

```diff
@@ -79,6 +79,13 @@ async function main(args: { context: vscode.ExtensionContext }): Promise<void> {
 		}
 		return
 	}
+	telemetry.capture({
+		event: "IDE-EXTENSION config loaded",
+		properties: {
+			vscode_version: vscode.version,
+			version: version,
+		},
+	})
```

Comment:
wie soll die config loaded sein, wenn du config erst 20 zeilen weiter unten eingelesen wird?

Reply:
hier https://github.com/inlang/inlang/blob/5946bece217ba6057ec97a0843586dec2e58652c/source-code/ide-extension/src/main.ts#L109 wird erst die config geladen?

Reply:
fair, bin da oben von was anderem ausgegangen aber wäre es dann nicht einfacher dieses event einfach in config loaded umzubennen? Weil das macht ja genau das, dieses Event liest die Config/ prüft ob die geladen ist. 

Oder in anderen worten. Was ist dann der Unterschied zwischen` CORE used config` und `IDE-Ex config is loaded`? 
<img width="598" alt="CleanShot 2023-07-17 at 17 28 51@2x" src="https://github.com/inlang/inlang/assets/72493222/43d4604e-571b-4363-aa01-d6e1ae312634">

Reply:
CORE used config sollte wahrscheinlich `ecosystem` used config heißen. Dadurch kann man plugin usage im ganzen ecosystem sehen. 

Füg einfach noch ein event `IDE-EXTENSION inlang config loaded` hinzu :D